Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/OnSelect.hs

```diff
@@ -309,13 +187,16 @@ mkPriceV2 quote =
       priceComputedValue = Nothing
     }
 
-mkItemTagsV2 :: DQuote.DriverQuote -> Spec.TagGroup
-mkItemTagsV2 quote =
-  Spec.TagGroup
-    { tagGroupDisplay = Just False,
-      tagGroupDescriptor = Just $ Spec.Descriptor (Just "general_info") (Just "General Info") Nothing,
-      tagGroupList = Just $ mkItemTagList quote
-    }
+mkItemTagsV2 :: DQuote.DriverQuote -> Bool -> Maybe Spec.TagGroup
+mkItemTagsV2 quote isValueAddNP
+  | not isValueAddNP = Nothing
+  | otherwise =
+    Just $
+      Spec.TagGroup
+        { tagGroupDisplay = Just False,
+          tagGroupDescriptor = Just $ Spec.Descriptor (Just "general_info") (Just "General Info") Nothing,
```

Comment:
Descriptor code needs to be updated with Tag enums

Reply:
fixed.